File: week5/simpleinherit.cpp
#include <iostream>
using namespace std;
class college
{
public:
    string collegeName = "MVGR";
    int collegeCode = 33;
};

class student : public college
{
public:
    string name = "Divya";
    int roll = 564;
};

int main()
{
    student obj;
    cout << "Student name : "<<obj.name << endl;
    cout << "Student roll num: "<<obj.roll << endl;
    cout << "College name: "<<obj.collegeName << endl;
    cout << "College code : "<<obj.collegeCode << endl;
}

File: week5/simpleinherit.java
class college {
    String collegeName = "MVGR";
    int collegeCode = 33;
}

class student extends college {
    String name = "Divya";
    int roll = 564;
}

class SimpleInherit {
    public static void main(String[] args) {
        student student1 = new student();
        System.out.println(student1.name);
        System.out.println(student1.roll);
        System.out.println(student1.collegeName);
    }
}
